locator/tablets.hh

```diff
@@ -158,6 +158,8 @@ enum class tablet_task_type {
     merge
 };
 
+static constexpr int64_t initial_sstables_repaired_at_after_merge = 1;
+
```

Comment:
I think it's less risky to use max(left, right).

Reply:
I think it is opposite. A bigger number is more risky because it has higher chance to mark unrepaired sstables as repaired.  To have a incremental repair only system, it is less risky to mark repaired sstables unrepaired, because unrepaired sstables will be picked by the next incremental repair.